models/api.models.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs/promises')
+
+function selectApi(){
+return fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8' ).then((result)=>{
+  return result
+})
```

Comment:
A good use of fs.readFile, but also keep in mind that in javascript you can require in a JSON file directly and access it that way, meaning that a readFile isn't strictly necessary here